README.md

```diff
@@ -30,6 +30,8 @@ Tasks
 | ExchangeName | string | Name of the exchange | sampleExchange |
 | RoutingKey | string | Routing key (as in RabbitMQ specification) | sampleQueue |
 | HostName | string | Address of the server hosting RabbitMQ | localhost or amqp://user:password@hostname:port/vhost |
+| WriteMessageCount | string | Amount of messages in the buffer which will be sent over messaging queue as a batch.  | 20 |
```

Comment:
If we are going to use the WriteMessageCount to signal when the batch should be committed, that should be reflected in the documentation. IMHO it would be more clear to have the separate task for committing the batch instead. This way the functionality isn't "magic" for the user